pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -204,6 +204,359 @@ type BMCDetails struct {
 	DisableCertificateVerification bool `json:"disableCertificateVerification,omitempty"`
 }
 
+// IDRACConfig the BIOS configuration in the iDRAC driver supports
+type IDRACConfig struct {
+	// Possible values: string value.
+	// +kubebuilder:validation:MaxLength=64
+	AssetTag string `json:"assetTag,omitempty"`
+
+	// Possible values: Bios, Uefi.
+	// +kubebuilder:validation:Enum="Bios";"Uefi"
+	BootMode string `json:"bootMode,omitempty"`
```

Comment:
This is being handled via a different field now.

Reply:
Thanks, done.

Reply:
I try to modify API, but i can't find `sriov` support in `irmc`, so i delete it from public bios settings.